markets/perps-market/contracts/storage/AsyncOrder.sol

```diff
@@ -148,6 +149,15 @@ library AsyncOrder {
         PerpsMarketConfiguration.Data storage marketConfig = PerpsMarketConfiguration.load(
             order.marketId
         );
+        uint256 sizeDeltaInUint256 = order.sizeDelta < 0
+            ? order.sizeDelta.mulDecimal(-1).toUint()
+            : order.sizeDelta.toUint();
+        if (marketConfig.maxMarketValue < perpsMarketData.size + sizeDeltaInUint256) {
```

Comment:
I think we need some additional logic here..
The `perpsMarketData.size` is currently storing the absolute value of all shorts and long. I think `maxMarketValue` is suppose to be a cap on either side, at least that's how it works on perps v2.

So I think we need to track `shortSize` and `longSize` in `PerpsMarket.sol`.  (I would also think it's clearer if we name them `shortOI` and `longOI`).
Maybe @sunnyvempati can shine in if this is suppose to work differently compared to preps v2.

Reply:
After talking to sunny, We can figure it out by looking at size + skew. see https://github.com/Synthetixio/synthetix-v3/blob/main/markets/perps-market/contracts/storage/PerpsMarket.sol#L226

Reply:
cool thanks! Will have a look!